teams/templates/includes/project.html

```diff
@@ -21,10 +21,13 @@ <h3>Project: {{ team.project.display_name}}</h3>
             {% else %}
                 <img src="{% static 'img/ci-hackathon--square.png' %}" width="100%" />
             {% endif %}
-            {% if request.user in team.participants.all %}
-            <a role="button" class="btn btn-ci mt-2" href="project/">Edit Project</a>
-            {% endif %}
         </div>
+        {% if request.user in team.participants.all and not showcase %}
+        <a role="button" class="btn btn-ci mt-2 w-100" href="{% url 'create_project' team_id=team.id %}">Edit Project</a>
+        <a role="button" class="btn btn-ci mt-2 w-100" href="{% url 'create_or_update_showcase' team_id=team.id %}">
+            {% if team.get_showcase %}Edit{% else%}Add{% endif%} Showcase
```

Comment:
formatting, either "with" or "without" spaces...
`{% else %}` or `{%else%}`
`{% endif %}` or `{%endif%}`
(first of both, more preferred, with the spaces)

Reply:
Just a typo changed it to `{% else %}` and `{% enfif %}`

Reply:
Hopefully `{% endif %}` instead of `{% enfif %}`